terra-jupyter-aou/Dockerfile

```diff
@@ -77,13 +88,64 @@ RUN mkdir -p /tmp/plink && \
   cd $HOME && \
   rm -rf /tmp/plink
 
+# GATK install
+ENV GATK_VERSION 4.1.4.1
```

Comment:
would it make sense to just base the aou image off of gatk one?

Reply:
Just a note GATK 4.2.0.0 is out, we have a ticket to update `terra-jupyter-gatk` as well:

https://github.com/broadinstitute/gatk/releases

Reply:
Oh wow, I didn't notice that the GATK image already included R and Python (I guess our image was effectively forked from there, a while back). I switched to gatk as the base and greatly the AoU Dockerfile, great suggestion!

I think we're fine to just accept the upgrade whenever it happens, the base version is fine for now (I don't think anyone is clamoring for anything specific from gatk for the AoU genomics demo project phase).